config/sentry.php

```diff
@@ -44,6 +47,12 @@
         // Capture views as spans
         'views' => true,
 
+        // Capture Redis operations as spans (this enables Redis events in Laravel)
+        'redis_commands' => env('SENTRY_TRACE_REDIS_COMMANDS', false),
```

Comment:
Disabled by default since it enables events on the Redis connections (which are disabled by default) but maybe there is no reason to be THAT safe and we can just enable it (and allow users to disable it if they don't want it)? We could also enable it by default and have the user do `Redis::enableEvents()` in their code to fully enable the feature.